File: src/main/java/no/novari/configuration/Application.java
package no.novari.configuration;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.data.jpa.repository.config.EnableJpaAuditing;

@SpringBootApplication(scanBasePackages = {
        "no.novari.configuration",
        "no.fintlabs"
})
@EnableJpaAuditing(auditorAwareRef = "tokenAuditorAware")
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);

    }

}

File: src/main/kotlin/no/novari/Application.kt
package no.novari

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.data.jpa.repository.config.EnableJpaAuditing

@SpringBootApplication
@EnableJpaAuditing(auditorAwareRef = "tokenAuditorAware")
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
